pkgs/by-name/be/bemoji/package.nix

```diff
@@ -10,8 +10,8 @@ stdenvNoCC.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "marty-oehme";
     repo = "bemoji";
-    rev = version;
-    hash = "sha256-XXNrUaS06UHF3cVfIfWjGF1sdPE709W2tFhfwTitzNs=";
+    rev = "refs/tags/v${version}";
```

Comment:
Isn't `rev = "v${version}"` enough?

Reply:
Only until a branch `v${version}` does get created.

In the python packages it seems to be the prefered way, and I try to make it a good habbit of mine.

Reply:
Good to know, thanks for the explanation.